drivers/at86rf2xx/at86rf2xx_internal.c

```diff
@@ -116,6 +116,16 @@ uint8_t at86rf2xx_get_status(const at86rf2xx_t *dev)
             & AT86RF2XX_TRX_STATUS_MASK__TRX_STATUS);
 }
 
+bool at86rf2xx_is_busy(const at86rf2xx_t *dev)
+{
+    uint8_t s;
+    while ((s = at86rf2xx_get_status(dev)) == AT86RF2XX_STATE_IN_PROGRESS) {}
```

Comment:
here you could treat "STATE_IN_PROGRESS" as busy. In that case you could simply add this case to the `if` below

Reply:
Alright, @maribu suggested the same.